src/modules/uavcan/uavcan_servers.hpp

```diff
@@ -88,7 +95,7 @@ class UavcanServers
 	static constexpr unsigned QueuePoolSize =
 		(NumIfaces * uavcan::MemPoolBlockSize * MaxCanFramesPerTransfer);
 
-	static constexpr unsigned StackSize  = 3500;
+	static constexpr unsigned StackSize  = 6000;
```

Comment:
Could you please rebase on master (I enabled -Os) and re-check stack sizes? We're just about out of RAM, so the few K here matter a lot.